### ISSUE TITLE
A new warning : Too many personality routines for compact unwind to encode 

### ISSUE THREAD

**Issue by finscn** [phoboslab/Ejecta#480]
I do merge with  upstream . Then  There is a new warning:

```
Too many personality routines for compact unwind to encode
```
What's mean ?
![2015-04-13 8 55 22](https://cloud.githubusercontent.com/assets/288367/7115938/bb1db5b6-e21f-11e4-9f0f-8f1ec69cc0ea.png)

**Comment by homma-ge** [comment]
me too, on iOS simulater.

**Comment by phoboslab** [comment]
Google suggests you can safely ignore this:
http://stackoverflow.com/questions/21150223/ld-warning-too-many-personality-routines-for-compact-unwind-to-encode

**Comment by finscn** [comment]
I found  if I use the  early iPhone  ( 4 / 4s / 5 )  simulator , there will be this warning.
Use iPhone 5s 6 6P , everything will be ok.

**Comment by michaKFromParis** [comment]
I bumped into this issue as well when trying to run on the iOS simulator while my code was running correctly on the device. In my case, it was not a warning but a linker error.

Luckily, i remembered that I added two flags for luajit to run correctly following the instructions of this page under the section Embedding LuaJIT:

-pagezero_size 10000 -image_base 100000000

My guess is that the image_base address is simply out of bounds on the host CPU.

Anyway, removing these settings made it work in my configuration.

So go to your project's settings and look for any hard wired values of this kind if not the same.

Hope this helps,

M
